console/console-label-conversion.any.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+// https://console.spec.whatwg/org/#counting
+// https://console.spec.whatwg/org/#timing
+
+// toString() console label test strings
+const toStringAssertionEnding = 'must call toString() on label when label is an object';
+const toStringTestNameEnding = 'label gets converted to string via label.toString() when label is an object';
+
+// toString error re-throwing console test strings
+const toStringRethrowAssertionEnding = 'must re-throw any exceptions thrown by label.toString() conversion';
+const toStringRethrowTestNameEnding = 'throws exceptions generated by erroneous label.toString() conversion';
```

Comment:
Not sure why these test strings are factored out instead of inlined; it makes things a big harder to read.

Reply:
Ah I think before using the for loop I had factored them out to avoid duplication, but that's not an issue at all now.